public/doc/openapi-3-v1.0.0.json

```diff
@@ -3722,17 +3813,8 @@
       "TagItem": {
         "type": "object",
         "properties": {
-          "name": {
-            "example": "architecture",
-            "readOnly": true,
-            "type": "string"
-          },
-          "namespace": {
-            "example": "openshift",
-            "readOnly": true,
-            "type": "string"
-          },
-          "value": {
+          "tag": {
```

Comment:
@lindgrenj6 Should this have readOnly: true since its being sent from the client. The same issue happens in topo PR. If you generate a client you would not be able to set the :tag attribute since its readonly

Reply:
@lindgrenj6 topo has fixed this issue they have removed the readOnly